packages/create-plugin/templates/datasource/provisioning/datasources/datasources.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1
+
+datasources:
+  - name: '{{ pluginName }}'
+  - type: '{{ normalize_id pluginName orgName 'datasource' }}'
+    access: proxy
+    isDefault: true
+    orgId: 1
+    version: 1
+    editable: true
+    uid: U9sW5bank
```

Comment:
Let's use a random generated UID here.

Reply:
I have a feeling you can omit the uid from the datasource provisioning. For any associated dashboards that reference the datasource I think if uid is missing it falls back to using the datasource name. 🤔 

```
// datasources.yml
apiVersion: 1
datasources:
  - name: TestData DB
    type: testdata
```

```
// panels.json
"targets": [
  {
    "csvContent": "Time,series,value,group\n1687272258,A,1,C\n1687272318,A,1,C\n1687272378,A,,C\n1687272438,A,2,C\n1687272498,A,2,C\n1687272258,B,3,C\n1687272318,B,3,C\n1687272378,B,3,C\n1687272438,B,3,C\n1687272498,B,3,C",
    "datasource": "TestData DB",
    "refId": "A",
    "scenarioId": "csv_content"
  }
],
```